### ISSUE TITLE
Migrate relevant settings to settings API

### ISSUE THREAD

**Issue by mdroidian** [dvargas92495/roam42#402]
# Reason
Better user experience.

# Examples

**Command Palette Plus**
```
To remove the confirmation modal, create a block somewhere in your graph with the text: #42Setting workBenchDcpConfirm off
```
**Daily Note Popup**
```
To toggle this feature off, create a block somewhere in your graph with the text: #42Setting dailySubtitles off and refresh Roam.
```
**Live Preview**
```
You can edit the delay time for when the Live Preview appears. To do so, add the following block text to any block in your graph:

#42Setting LivePreviewDelay 100
```
etc

**Comment by dvargasfuertes** [comment]
Yup - this was all in the "I'll deal with it after it's in Roam Depot" bucket. Adding to backlog

**Comment by mdroidian** [comment]
With all of the feature/modules, the Roam settings page is already quite long.  This requires the user to scroll to get to any settings.  Not to mention all the hotkeys that would be added when `Command Palette+` / `HotKeys` is toggled.

I'm exploring different options to handle this.

# Remove Description from Feature Toggles
Don't really like this option.  Doesn't seem to reduce enough space.  User would have to know to scroll to find descriptions.
![image](https://github.com/RoamJS/workbench/assets/3792666/c88a6f1e-1d9e-429a-83ed-33f282cb9039)

# Custom React Table
Better.  But the "Name" of the setting takes up a lot of space
![image](https://github.com/RoamJS/workbench/assets/3792666/bfa687be-c454-4e3c-8d49-9c99ee60465c)

# Settings in page
I'm kind of a fan of what's been done with `roam/js/weekly-notes`
![image](https://github.com/RoamJS/workbench/assets/3792666/8aba449b-a6e5-457e-abaa-f9096b69aca6)

We could update the settings descriptions to link to these pages

**Comment by mdroidian** [comment]
For context, here are the settings for each feature/module I noted

![image](https://github.com/RoamJS/workbench/assets/3792666/ab675a12-901d-42d0-aec2-963e564ee1a9)
